### PR TITLE
[RHACS] Fix PDF button

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Changes:
- `_stylesheets/docs.css` copied from the `main` branch.
- `_stylesheets/print.css` new file, copied from the `main` branch.
- `_templates/_page_openshift.html.erb` copied from the `main` branch and replaced getPDF function for `openshift-acs` distribution with Print PDF option.

Based on the changes in the following PRs:
- https://github.com/openshift/openshift-docs/pull/49287
- https://github.com/openshift/openshift-docs/pull/49194

Cherrypick into:
- `rhacs-docs-3.74`
- `rhacs-docs-3.73`
- `rhacs-docs-3.72`

### CODE REVIEW

File: _templates/_page_openshift.html.erb [hunk 1]
```diff
@@ -67,12 +58,32 @@
 
       <span>
         <div class="alert alert-danger" role="alert" id="support-alert">
-          <strong>You are viewing documentation for a release that is no longer supported.</strong> The latest supported version of version 3 is <a href="https://docs.openshift.com/container-platform/3.11/welcome/index.html" class="link-primary" style="color: #545454 !important;">[3.11]</a>. For the most recent version 4, see <a href="https://docs.openshift.com/container-platform/latest/welcome/index.html" style="color: #545454 !important" class="link-primary">[4]</a>.
+          <strong>You are viewing documentation for a release that is no longer maintained.</strong> The latest supported version of version 3 is <a href="https://docs.openshift.com/container-platform/3.11/welcome/index.html" class="link-primary" style="color: #545454 !important;">[3.11]</a>. For the most recent version 4, see <a href="https://docs.openshift.com/container-platform/latest/welcome/index.html" style="color: #545454 !important" class="link-primary">[4]</a>.
```
Comment (aireilly) [inline _templates/_page_openshift.html.erb:61]: This looks odd - why is this change here? This is not part of the PDF generation updates.

File: _templates/_page_openshift.html.erb [hunk 2]
```diff
@@ -67,12 +58,32 @@
...
         </div>
       </span>
 
       <% end %>
 
+      <% if ((unsupported_versions.include? version) && (distro_key == "openshift-acs")) %>
```
Comment (aireilly) [inline _templates/_page_openshift.html.erb:67]: Same question as above - this is not part of the PDF updates.

**Review by aireilly — COMMENTED** [review]
Changes LGTM generally and PDF feature verified with a local build.  

There are are some changes in here that are unrelated to the PDF feature. Probably missed a cherrypick from some previous work?

**Comment by gaurav-nelson** [comment]
Thank you @aireilly 

As I mentioned in the description, `_templates/_page_openshift.html.erb` I copied the file from the `main` branch and replaced getPDF function for `openshift-acs` distribution with Print PDF option.

File: _templates/_page_openshift.html.erb (hunk 2 above)
Comment (gaurav-nelson) [reply _templates/_page_openshift.html.erb:67]: `_templates/_page_openshift.html.erb` I copied this file from the `main` branch.

File: _templates/_page_openshift.html.erb (hunk 1 above)
Comment (gaurav-nelson) [reply _templates/_page_openshift.html.erb:61]: `_templates/_page_openshift.html.erb` I copied this file from the `main` branch.

**Comment by aireilly** [comment]
LGTM
